src/model/webhook.rs

```diff
@@ -234,6 +234,15 @@ impl Webhook {
             Err(why) => Err(why),
         }
     }
+
+    /// Returns the url of the webhook.
+    ///
+    /// ```rust,no_run
```

Comment:
```suggestion
    /// ```rust,ignore
```
`no_run` tells rustdoc to disallow running the doctest, but still compile it. Obviously from CI, your example fails because `hook` is a nonexistent variable. The `ignore` directive tells rustdoc to not compile the doctest at all, so you wouldn't have to worry about errors.

If you don't like this suggestion, then there's another method that is already employed in this file. Create a fake `Webhook` instance using `Http`:
```suggestion
    /// ```rust,no_run
    /// # use serenity::http::Http;
    /// # use std::sync::Arc;
    /// #
    /// # let http = Arc::new(Http::default());
    /// let id = 245037420704169985;
    /// let token = "ig5AO-wdVWpCBtUUMxmgsWryqgsW3DChbKYOINftJ4DCrUbnkedoYZD0VOH1QLr-S3sV";
    /// 
    /// let hook = http.as_ref().get_webhook_with_token(id, token)
    ///     .expect("valid webhook");
```